webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -85,9 +85,7 @@ class Layout extends React.Component {
           onItemClick={primary =>
             handleMenuClick(primary, activeMenu, changeActiveMenu)
           }
-          onNavigate={({ href, preventHref }) =>
-            history.push(href, { useTurbolinks: preventHref })
-          }
+          onNavigate={({ href }) => history.push(href)}
```

Comment:
do we actually need to use `onNavigate` that basically does the same as just letting the href do it's thing?

Reply:
Isn't it here so it will use the react-router `history` object?

Reply:
good point, i was confusing it with the `window.history`